### DISCUSSION TITLE
Change the color of the ego car.

### DISCUSSION THREAD

**Discussion by LianhaoYin** [eleurent/highway-env#253] [Q&A]
In multiple-ego-car situations (controlled_vehicles >= 2, highway environment) , is it possible to change the color of one of the ego cars? Could you give some hints on how to do such a change? Thanks! @eleurent

**Comment by eleurent** [comment]
Hi @LianhaoYin 
It is not implemented at the moment, but I agree that it's a good idea!
I guess the easiest would be to simply fill the `color` field of controlled vehicles when they are created.
Once the field is set, it will be used by default, see [the code for choosing a color for vehicle rendering](https://github.com/eleurent/highway-env/blob/master/highway_env/vehicle/graphics.py#L169).
